src/libs/actions/Session/index.js

```diff
@@ -90,11 +90,29 @@ function signOutAndRedirectToSignIn() {
  * @param {String} [login]
  */
 function resendValidationLink(login = credentials.login) {
-    Onyx.merge(ONYXKEYS.ACCOUNT, {isLoading: true});
-    DeprecatedAPI.ResendValidateCode({email: login})
-        .finally(() => {
-            Onyx.merge(ONYXKEYS.ACCOUNT, {isLoading: false});
-        });
+    const optimisticData = [{
+        onyxMethod: CONST.ONYX.METHOD.MERGE,
+        key: ONYXKEYS.ACCOUNT,
+        value: {isLoading: true},
```

Comment:
```suggestion

        value: {
            isLoading: false,
            errors: null,
            message: '',
        },
```

I think you should clear the success message and the errors as soon as you click.